data/incipias/incipias ships.txt

```diff
@@ -0,0 +1,239 @@
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+ship "Venta"
+	"plural" "Ventae"
+	"sprite" ship/venta
+	"thumbnail" thumbnail/venta
+	"swizzle" 0
+	attributes
+		"category" "Heavy Warship"
+		licenses
+			"Hicemus Conflict"
+		"cost" 25000000
+		"hull" 12000
+		"shields" 16000
+		"required crew" 40
+		"bunks" 45
+		"mass" 1200
+		"drag" 6.
+		"heat dissipation" .7
+		"cargo space" 20
+		"outfit space" 552
+		"weapon capacity" 240
+		"engine capacity" 130
+		"gaslining" 1
+		"fuel capacity" 200
+	outfits
+		"Solid-Class Shield"
+		"Liquid-Class Shield"
+		"Liquid Hydrogen Cell"
+		"Plasma Discharger" 2
+		"Star Tail" 4
+		"Star Tail Cell" 80
+		"Star Tail Storage" 2
+		"Heavy Machine Gun" 40
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+	engine -90.5 42
+	engine 0 42
+		"over"
+	engine 90 42
+	gun 58.5 -76.5
+	gun 29 -76.5
+	gun 29 -76.5
+	gun -28 -76.5
+	gun -28 -76.5
+	gun -57 -76
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "When the Hicemus came to power they needed a ship to symbolize their strength. With six guns and one turret, the Venta is the most powerful ship in Incipias history. Thanks to its sleek and aerodynamic design it is able to hunt down outlaws in space as well as in the clouds of a gas giant."
+
+ship "Nimbus"
+	"plural" "Nimbi"
+	"sprite" ship/nimbus
+	"thumbnail" thumbnail/nimbus
+	"swizzle" 0
+	attributes
+		"category" "Medium Warship"
+		licenses
+			"Hicemus Conflict"
+		"cost" 500000
+		"hull" 5000
+		"shields" 7500
+		"required crew" 20
+		"bunks" 40
+		"mass" 900
+		"drag" 7
+		"heat dissipation" .7
+		"cargo space" 75
+		"outfit space" 450
+		"weapon capacity" 120
+		"engine capacity" 85
+		"gaslining" 1
+		"fuel capacity" 200
+	outfits
+		"Liquid-Class Shield"
+		"Liquid Hydrogen Cell"
+		"Plasma Discharger" 4
+		"Stagnation Beam"
+		"Heavy Machine Gun" 20
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+	engine 0 120
+	engine 0 120
+	engine -30.5 120
+	engine 30.5 120
+	gun -15 -117
+	gun 15 -117
+	gun 0 -117
+	gun 0 -117
+	turret 0 31.5
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "The Nimbus is the first spaceship that the Incipias built that was really able to navigate through a system. Since it was first manufactured it has been loved among wealthier captains and a dream of captains just starting out."
+	description "Although the Nimbus is labeled as a medium warship, it is a multipurpose ship. Whether one uses it for mining, passengers, or cargo, it will perform its job nicely. It is only occasionally used in combat because the Incipias are not currently fighting open wars."
```

Comment:
```suggestion
	description "Although the Nimbus is labeled as a medium warship, it is a multipurpose ship. Whether one uses it for mining, passenger transport or cargo, it will perform its job nicely. It is only occasionally used in combat because the Incipias are not currently fighting open wars."
```